pkg/reconciler/vertex/controller.go

```diff
@@ -251,20 +251,6 @@ func (r *vertexReconciler) reconcile(ctx context.Context, vertex *dfv1.Vertex) (
 		if needToCreate {
 			labels := map[string]string{}
 			annotations := map[string]string{}
-			if pipeline.Spec.Templates != nil && pipeline.Spec.Templates.VertexTemplate != nil {
-				apt := pipeline.Spec.Templates.VertexTemplate.AbstractPodTemplate
-				apt.ApplyToPodSpec(podSpec)
```

Comment:
who does the apply now?

Reply:
Do not need to apply anymore, the spec comes from vertex already has everything.